src/Controls/src/Core/Compatibility/Handlers/NavigationPage/iOS/NavigationRenderer.cs

```diff
@@ -714,6 +720,24 @@ void UpdateBarBackground()
 			}
 		}
 
+		void ApplyTransparentBarBackground()
+		{
+			if (OperatingSystem.IsIOSVersionAtLeast(13) || OperatingSystem.IsMacCatalystVersionAtLeast(13))
+			{
+				var navBar = new UINavigationBarAppearance();
+				navBar.ConfigureWithTransparentBackground();
+				NavigationBar.CompactAppearance = NavigationBar.StandardAppearance = NavigationBar.ScrollEdgeAppearance = navBar;
+			}
+			else
+			{
+				NavigationBar.SetBackgroundImage(new UIImage(), UIBarMetrics.Default);
```

Comment:
Can we consolidate this code with the same code in shell? It seems like it's pretty much the same?